WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt

```diff
@@ -132,13 +172,46 @@ class PostListFragment : Fragment(),
         dispatcher.register(this)
     }
 
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(outState)
+        outState.putSerializable(WordPress.SITE, site)
+        rvScrollPositionSaver.onSaveInstanceState(outState, recyclerView)
+        outState.putIntArray(KEY_TRASHED_POST_LOCAL_IDS, trashedPostIds.map { it.first }.toIntArray())
+        outState.putLongArray(KEY_TRASHED_POST_REMOTE_IDS, trashedPostIds.map { it.second }.toLongArray())
+        outState.putLongArray(KEY_UPLOADED_REMOTE_POST_IDS, uploadedPostRemoteIds.toLongArray())
+    }
+
     override fun onDestroy() {
         EventBus.getDefault().unregister(this)
         dispatcher.unregister(this)
 
         super.onDestroy()
     }
 
+    override fun onResume() {
+        super.onResume()
+
+        // scale in the fab after a brief delay if it's not already showing
+        if (fabView?.visibility != View.VISIBLE) {
+            val delayMs = resources.getInteger(R.integer.fab_animation_delay).toLong()
+            Handler().postDelayed({
+                if (isAdded) {
+                    AniUtils.scaleIn(fabView, AniUtils.Duration.MEDIUM)
+                }
+            }, delayMs)
+        }
+    }
+
+    override fun onDetach() {
```

Comment:
This method is unchanged, it's moved for better organization. Here is the version from base branch for easy comparison: https://github.com/wordpress-mobile/WordPress-Android/blob/feature/list-store-integration-for-post-list-master/WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt#L329-L337